src/main.ts

```diff
@@ -111,5 +111,27 @@ export function configure(aurelia: Aurelia): void {
         notificationService.showNonDisappearingNotification(NotificationType.ERROR, errorMessage);
       });
     }
+
+    if (applicationRunsInElectron) {
+      const ipcRenderer: any = (window as any).nodeRequire('electron').ipcRenderer;
+      const notificationService: NotificationService = aurelia.container.get('NotificationService');
+
+      ipcRenderer.send('app_ready');
+
+      ipcRenderer.on('update_error', (event: any) => {
+        notificationService.showNotification(NotificationType.INFO, 'Update Error');
+      });
+
+      ipcRenderer.on('update_available', (event: any) => {
+        notificationService.showNotification(NotificationType.INFO, 'Update available, started downloading.');
+      });
+
+      ipcRenderer.on('update_downloaded', (event: any) => {
+        const installButton: string = `<a href="javascript:nodeRequire('electron').ipcRenderer.send('quit_and_install')">here</a>`;
+
+        notificationService.showNonDisappearingNotification(NotificationType.INFO, `Update ready! Click ${installButton} to restart and install!`);
```

Comment:
Maybe use `Update ready` as headline:

```suggestion
        const messageTitle: string = '<h4>Update ready!</h4>';
        const messageBody: string = `Click ${installButton} to restart and install!`;

        notificationService.showNonDisappearingNotification(NotificationType.INFO, `${messageTitle}\n${messageBody}`);
```